packages/blade/src/components/Input/BaseInput/BaseInputTagSlot.web.tsx

```diff
@@ -10,7 +10,7 @@ import { useIsMobile } from '~utils/useIsMobile';
 import { MetaConstants } from '~utils/metaAttribute';
 import { size as sizeToken } from '~tokens/global';
 
-const MINUMUM_INPUT_SPACE = 30;
+const MINUMUM_INPUT_SPACE = 60;
```

Comment:
Expected to be changed?

Reply:
yup. The typing area left is too small when input is filled with tags. This will improve the experience for AutoComplete as well